package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "MongoDB Compass",
   "description": "The MongoDB GUI",
   "homepage": "https://compass.mongodb.com",
-  "version": "1.21.0-dev",
+  "version": "1.21.0-dev.1",
```

Comment:
?

Reply:
Migration wouldn't run in dev without this since the last seen version of the app would be equal to the current version.